common/pgbackup/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ .Release.Name }}-pgbackup"
+  annotations:
+    {{- if and $.Values.global.linkerd_enabled $.Values.global.linkerd_requested }}
+    linkerd.io/inject: enabled
+    {{- end }}
+spec:
+  selector:
+    name: "{{ .Release.Name }}-pgbackup"
+  ports:
+  - name: pgbackup
+    port: 5432
```

Comment:
This should probably be 8080 to match the container port. 5432 is for the postgres binary protocol.